src/Proto.Actor/EventStream.cs

```diff
@@ -56,9 +56,18 @@ public Subscription<T> Subscribe(Action<T> action)
 
         public void Publish(T msg)
         {
+            var logger = Log.CreateLogger<EventStream>();

```

Comment:
Actually EventStream inherits EventStream<T>. And maybe it would be reasonable to have protected logger which was iniated in the constructor of the EventStream<T>. But I think that if you as user inherits EventStream and see protected logger it's a little bit strange..  My suggestion is to have different loggers in EventStream and EventStream<T>. What do you think?